srcpkgs/python3-pytest-xdist/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python3-pytest-xdist'
+pkgname=python3-pytest-xdist
+version=2.5.0
+revision=1
+wrksrc="pytest-xdist-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip python3-wheel"
```

Comment:
```suggestion
hostmakedepends="python3-setuptools"
```
pip just hides dependency issues, please fix this properly without pip

Reply:
Thanks @paper42.  I think I misread the error message when I was working on this before.  I think I found the correct dependency this time.